lib/fs.js

```diff
@@ -207,6 +207,15 @@ fs.accessSync = function(path, mode) {
   binding.access(pathModule._makeLong(path), mode);
 };
 
+fs.accessibleSync = function(path, mode) {
+  try {
+    fs.accessSync(path, mode);
```

Comment:
I'd lie to think there was a specific reason I suggested this, but can't think of it.